### PR TITLE
do lesson

### PR DESCRIPTION
**Pull request by dogiadat**

### CODE REVIEW

File: app/controllers/admin/words_controller.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Admin::WordsController < Admin::BaseController
   before_action :load_word, only: [:edit, :update, :destroy]
 
   def index
-    @categories = Category.all
+    @categories = Category.order(:name)
     word_type = params[:word_type] || Settings.all_word
     @words = Word.send(word_type, current_user)
      .by_category(params[:by_category])
```
Comment (HoangLanh) [inline app/controllers/admin/words_controller.rb]: ident

File: app/controllers/words_controller.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@ class WordsController < ApplicationController
   before_action :logged_in_user
 
   def index
-    @categories = Category.all
+    @categories = Category.order(:name)
     word_type = params[:word_type] || Settings.all_word
     @words = Word.send(word_type, current_user)
      .by_category(params[:by_category])
```
Comment (HoangLanh) [inline app/controllers/words_controller.rb]: ident

File: app/assets/stylesheets/custom.css.scss [hunk 3]
```diff
@@ -72,10 +72,24 @@ h1 {
...
-  margin-bottom :10px
+  margin-bottom :10px;
+  padding: 10px;
 }
 .users.follow {
   padding: 0;
 }
+
+table, td, th {
+    border: 1px solid #ddd;
```
Comment (HoangLanh) [inline app/assets/stylesheets/custom.css.scss]: ident

File: app/assets/stylesheets/custom.css.scss [hunk 4]
```diff
@@ -72,10 +72,24 @@ h1 {
...
+    text-align: left;
+}
+
+table {
+    border-collapse: collapse;
```
Comment (HoangLanh) [inline app/assets/stylesheets/custom.css.scss]: ident

File: app/assets/stylesheets/custom.css.scss (hunk 3 above)
Comment (HoangLanh) [inline app/assets/stylesheets/custom.css.scss]: ident

File: app/assets/stylesheets/custom.css.scss [hunk 5]
```diff
@@ -72,10 +72,24 @@ h1 {
...
+}
+
+th, td {
+    padding: 15px;
```
Comment (HoangLanh) [inline app/assets/stylesheets/custom.css.scss]: ident

**Comment by HoangLanh** [comment]
ok

File: app/views/lessons/show.html.erb [hunk 6]
```diff
@@ -1,38 +1,76 @@
...
+          <%= @lesson.lesson_words.correct_answer.count %>
+            / <%= @lesson.lesson_words.count %>
+        </strong>
+        <%= form_for @lesson do |f| %>
+          <%= f.hidden_field :is_completed, value: false %>
+          <%= f.fields_for :lesson_words do |builder| %>
+            <div class="lesson_form">
+              <strong><%= builder.object.word.content %></strong>
+              <br/>
+                <div>
```
Comment (nhunght) [inline app/views/lessons/show.html.erb]: indent

**Pull request merged**